varlink/connection.go

```diff
@@ -122,30 +122,114 @@ func (c *Connection) Send(method string, parameters interface{}, flags uint64) (
 	}
 
 	b = append(b, 0)
-	_, err = c.Writer.Write(b)
+	_, err = c.conn.Write(context.Background(), b)
```

Comment:
Why so many context.Background() in this function instead of passing in a context?

Reply:
These methods are only kept for backwards compatibility. Note the `XContext` methods.